engine/emoji-picker

```diff
@@ -16,8 +16,8 @@
 #  You should have received a copy of the GNU General Public License
```

Comment:
@mike-fabian alright i will get in touch with a designer and see if he can do some better icons based on the original design. But for now i will generate the png icons from svg ones and remove the symbolic icon so this PR can be merged